src/task17/task17

```diff
@@ -0,0 +1,44 @@
+const box = {
+    locked: true,
+    unlock() { this.locked = false; },
+    lock() { this.locked = true; },
+    _content: [1, 2, 3],
+    get content() {
+        if (this.locked) throw new Error("Locked!");
+        return this._content;
+    }
+};
+
+const withBoxUnlocked = body => {
+    let isLocked = box.locked;
```

Comment:
isLocked does not make code easy, my opinion

Reply:
you want it to be wasLocked, don't you?